Functions/BIDS/xASL_bids_MergeNifti_Merge.m

```diff
@@ -130,19 +151,14 @@
                 catch
                     fprintf('Sorting failed...\n');
                 end
-            end
+			end
+			
             % Write changes to JSON
-            structFirstJSON = spm_jsonread(firstJSON);
-            structFirstJSON.EchoTime = EchoTimes;
-            spm_jsonwrite(firstJSON, structFirstJSON);
-        end
-        % Copy the first JSON to this name
-        if ~isempty(firstJSON)
-            xASL_Copy(firstJSON,fullfile(Fpath,[nameMerged '.json']),1);
-        end
+			outputJSON.EchoTime = EchoTimes;
```

Comment:
Why is this dealt with separately? Shouldn't other vectors be dealt with separately as well then (PLDs, labdur) for multi-PLD/Hadamard?

Reply:
Because it needs to be dealt with separately. We need to assemble TE files from several JSONs. PLDs and labdurs are never in a separate file for multi-PLD and Hadamard.

Once I have such a flavor, I can write and tweak such fix, but so far, it is only needed for TE. 
And I dont' want to write a general fix, because it is unclear if this would work in a future case (as unfortunately, things are not really done consistently in DICOM that would allow that).

Reply:
I would want to generalize this. You could just add this for PLDs and labeling duration (which are the only ones where we need to know this right). If they are always the same, we don't care (we take unique) but if they differ, we issue a warning that differences are detected and imported as ...multi-TE or multi-PLD or whatsoever.

So you are saying that you don't want unexpected behavior: I say that we issue a warning (so we will at least know this), and IF there are multiple PLDs between NIfTIs which could make sense, then we catch them at least. Anyway, choose what you want and resolve this.

Reply:
Warning added for PLD and LabDur. But only warning - concatenation has to be tested once we have such a dataset.